deepquest/__main__.py

```diff
@@ -33,6 +40,19 @@ def score(args):
     from . import score
     score(args.files)
 
+def set_gpu_id(gpuid):
+    import os
+    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+    gpuid = gpuid.split(',') if ',' in gpuid else gpuid.split()
+    gpustr = ''
+    gpucount = 0
+    for g in gpuid:
+        gpustr += str(g).strip() + ','
+        gpucount += 1
+    gpustr = gpustr[0:-1]
```

Comment:
You can avoid the `if` by using an `re` to split on anything that's not a word:
    
    gpu_list = re.split(r"\W+", gpuid)

So this would allow both `0,2,3` and `0 2 3` as well as more weird things like `0 2#3`.

Your `for` loop is the same as `",".join`:

    gpustr = ",".join(gpu_list)
    gpucount = len(gpu_list)

Reply:
4443cbc will do this. I like this.